File: tools/bench/answers/go_haiku/s3/15_wordlens.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	sentence := "pack my box with five dozen liquor jugs"
	words := strings.Fields(sentence)
	var longest string
	for _, word := range words {
		if len(word) > len(longest) {
			longest = word
		}
	}
	totalLen := 0
	for _, word := range words {
		totalLen += len(word)
	}
	avgLen := totalLen / len(words)
	fmt.Println(longest)
	fmt.Println(avgLen)
}

File: tools/bench/answers/python_haiku/s2/15_wordlens.py
s = "pack my box with five dozen liquor jugs"
words = s.split()
longest = words[0]
for word in words:
    if len(word) > len(longest):
        longest = word
total_len = sum(len(word) for word in words)
avg_len = total_len / len(words)
print(longest)
print(int(avg_len))
